google-account-plugin/src/com/google/cloud/tools/intellij/stats/TrackingEventBuilder.java

```diff
@@ -39,19 +39,13 @@
   }
 
   @Override
-  public FluentTrackingEventWithValue withLabel(String label) {
-    this.label = Preconditions.checkNotNull(label);
-    return this;
-  }
-
-  @Override
-  public PingsAnalytics setValue(Integer value) {
-    this.value = Preconditions.checkNotNull(value);
+  public FluentTrackingEventWithMetadata withMetadata(String key, String value) {
```

Comment:
not 'with'Metadata. It should be clear that it appends.  I'd rename to addMetadata.